tests/tools/oesign/test-enclave/enclave/enc.c

```diff
@@ -54,20 +68,55 @@ bool is_test_signed()
     return is_test_signed;
 }
 
+static void* _find_claim(
+    const oe_claim_t* claims,
+    size_t claims_size,
+    const char* name)
+{
+    for (size_t i = 0; i < claims_size; i++)
+    {
+        if (strcmp(claims[i].name, name) == 0)
+            return claims[i].value;
+    }
+    return NULL;
+}
+
+void formatted_string_from_uuid(
+    uint8_t* uuid_id,
+    size_t uuid_id_size,
+    char* formatted_str_id,
+    size_t formatted_str_id_size)
```

Comment:
```suggestion
    char* formatted_string,
    size_t formatted_string_size)
```
Avoid abbreviations, and match function name (which has "string" not "string id")

Reply:
Thanks for the review, I applied this suggestion to all 4 parameters of `formatted_string_from_uuid()`